lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -112,7 +112,7 @@ sub test_network_interface {
     #flag SRIOV test as it need not restart network service
     my $is_sriov_test = "false";
     $is_sriov_test = "true" if caller 0 eq 'sriov_network_card_pci_passthrough';
-    script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180) if (is_sle('=11-sp4') || ($routed == 1) || ($isolated == 1));
+    script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180);
```

Comment:
What does "-PN" mean ?

Reply:
I guess you just scan "open ssh" right ? Something like "wait for ssh up" ?

Reply:
What about restarting guest ? Can restart solves the problem ?

Reply:
> I guess you just scan "open ssh" right ? Something like "wait for ssh up" ?

sure, add this step just to check with guest ssh service as up status here

Reply:
> What about restarting guest ? Can restart solves the problem ?

No, I totally do not agree on this. 
1. here just only attach virtual network interface to via hotplug way, we'd better to keep guest system as running|online status
2. if try to use with "restart" or "reboot" guest, which will involve other problem, so "reboot" guest do not resolve current problem. 
Also, imho, during our (vir) testing,  we'd better to no restart(reboot) guest system, If it is not a necessary requirement.

Reply:
It seems "waiting" is more proper here. Yes.

Reply:
> What does "-PN" mean ?

-PN: (Continue)Scan under Firewall Protection

Reply:
> It seems "waiting" is more proper here. Yes.

Yep, before get `$nic` from attached virtual network interface during call `test_network_interface()`, we had ping with guest. 
now, we need to ensure guest system ssh service as UP status.